build.gradle

```diff
@@ -31,3 +43,25 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+detekt {
```

Comment:
It would be nice to generate a config file as well, and add to the configuration here.

Reply:
I oganized detekt, and sonarqube task into gradle files.